src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { useEffect, useState } from 'react'
+
+import { ViewerContainer } from '@amsterdam/arm-core'
+import type { Feature, FeatureCollection } from 'geojson'
+
+import { useFetch } from 'hooks'
+import configuration from 'shared/services/configuration/configuration'
+import MAP_OPTIONS from 'shared/services/configuration/map-options'
+import { MapMessage } from 'signals/incident/components/form/MapSelectors/components/MapMessage'
+import type { Bbox } from 'signals/incident/components/form/MapSelectors/hooks/useBoundingBox'
+
+import type { Filter, Point, Properties } from '../../types'
+import { FilterPanel } from '../FilterPanel'
+import { IncidentLayer } from '../IncidentLayer'
+import { getFilteredIncidents } from '../utils'
+import { Wrapper, Container, StyledMap } from './styled'
+
+export const IncidentMap = () => {
+  const [bbox, setBbox] = useState<Bbox | undefined>(undefined)
+  const [mapMessage, setMapMessage] = useState<string>('')
+  const [filters, setFilters] = useState<Filter[]>([])
+  const [filteredIncidents, setFilteredIncidents] =
+    useState<Feature<Point, Properties>[]>()
+
+  const { get, data, error } = useFetch<FeatureCollection<Point, Properties>>()
+
+  useEffect(() => {
+    if (!bbox) return
+
+    const { west, south, east, north } = bbox
+    const searchParams = new URLSearchParams({
+      bbox: `${west},${south},${east},${north}`,
+    })
+
+    get(`${configuration.GEOGRAPHY_PUBLIC_ENDPOINT}?${searchParams.toString()}`)
+  }, [get, bbox])
+
+  useEffect(() => {
+    if (!data) {
```

Comment:
consider just putting 44-45 in if(data). I would only early return if otherwise there would be a redundant calculation ed.